src/components/MapEmbed.js

```diff
@@ -16,20 +18,24 @@
  * @return {JSX.Element} The rendered React component.
  */
 const MapEmbed = (props) => {
-	const { location, zoom, type, className } = props;
-	let { height } = props;
+	const { zoom, type, className } = props;
+	let { location, height } = props;
 
 	if (!height) {
 		height = 300;
 	}
 
-	const style = { border: 0, height, width: '100%' };
-	const baseUrl = 'https://maps.google.com/maps';
+	if (!isSinglePostInEditor() && !location) {
+		location = '60 29th Street #343, San Francisco, CA 94110, USA';
```

Comment:
@mauteri Assuming this hard-coded address is a default location to use for the map?